cmd/boulder-ca/main.go

```diff
@@ -146,6 +146,9 @@ func main() {
 	defer logger.AuditPanic()
 	logger.Info(cmd.VersionString(clientName))
 
+	err = features.Set(c.PA.Features)
```

Comment:
This duplication and below of setting features is unnecessary, each binary should only do this once using the `features` field from the top level of its configuration object.